### PR TITLE
Adapt for elrond-go node

### PR DESCRIPTION
**Pull request by iulianpascalau**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CURRENT_DIRECTORY := $(shell pwd)
```
Comment (CostinCarabas) [inline Makefile]: Are variables CURRENT_DIRECTORY and TESTS_TO_RUN used?

File: pubsub.go [hunk 2]
```diff
@@ -23,8 +23,8 @@ import (
 	timecache "github.com/whyrusleeping/timecache"
 )
 
-// DefaultMaximumMessageSize is 1mb.
-const DefaultMaxMessageSize = 1 << 20
+// DefaultMaxMessageSize is 2MB
+const DefaultMaxMessageSize = 1 << 21
```
Comment (CostinCarabas) [inline pubsub.go:27]: Is there a reason why `DefaultMaxMessageSize` is not a typed variable?

Comment (CostinCarabas) [reply pubsub.go:27]: Also, there is a second comment that should be changed: `// messages. The default value is 1MiB (DefaultMaxMessageSize).`

File: Makefile (hunk 1 above)
Comment (iulianpascalau) [reply Makefile]: removed the whole file. No longer needed as we run the tests in teamcity environment

File: pubsub.go (hunk 2 above)
Comment (iulianpascalau) [reply pubsub.go:27]: If not specified, the default is int.
https://go.dev/play/p/IGETqnIyywz

**Review by bogdan-rosianu — DISMISSED** [review]

File: pubsub.go (hunk 2 above)
Comment (iulianpascalau) [reply pubsub.go:27]: changed the comment

**Review by bogdan-rosianu — DISMISSED** [review]

File: topicExtra.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	}
+
+	m := &pb.Message{
+		Data:  data,
+		Topic: &t.topic,
+		From:  nil,
+		Seqno: nil,
+	}
+	if t.p.signID != "" {
+		m.From = []byte(t.p.signID)
```
Comment (AdoAdoAdo) [inline topicExtra.go]: maybe don't accept nil signKey and you can remove this (keep seqno init though)

Comment (iulianpascalau) [reply topicExtra.go]: good catch. Will refactor

Comment (iulianpascalau) [reply topicExtra.go]: done

File: topicExtra.go [hunk 4]
```diff
@@ -2,14 +2,25 @@ package pubsub
...
 import (
 	"context"
+	"errors"
 
 	"github.com/libp2p/go-libp2p-core/crypto"
 	"github.com/libp2p/go-libp2p-core/peer"
 	pb "github.com/libp2p/go-libp2p-pubsub/pb"
 )
 
+var errNilSignKey = errors.New("nil sign key")
```
Comment (bogdan-rosianu) [inline topicExtra.go:12]: could have grouped them into:
```
var (
  ...
)
```

**Review by AdoAdoAdo — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**